src/foam/u2/wizard/agents/StepWizardAgent.js

```diff
@@ -77,7 +75,12 @@ foam.CLASS({
       };
 
       view.data = this.wizardController;
-      view.onClose = this.stack.back.bind(this.stack);
+      view.onClose = (() => {
+        if ( this.stack.BACK.isEnabled(this.stack.pos) )
+          this.stack.back();
+        else
+          wizardStackBlock.removed.pub();
```

Comment:
```suggestion
          // This is temporarily necessary to fake a StackBlock removal
          // in case the stack is empty when the wizard is pushed.
          wizardStackBlock.removed.pub();
```